types/chrome-remote-interface/index.d.ts

```diff
@@ -149,6 +149,14 @@ declare namespace CDP {
         on<T extends keyof ProtocolMappingApi.Events>(event: T, callback: (params: ProtocolMappingApi.Events[T][0], sessionId?: string) => void): void;
         // '<domain>.<method>.<sessionId>' i.e. Network.requestWillBeSent.abc123
         on(event: string, callback: (params: object, sessionId?: string) => void): void;
+        // client.send(method, [params], [sessionId], [callback])
+        send<T extends keyof ProtocolMappingApi.Commands>(event: T, callback: (returnMessage: ProtocolMappingApi.Commands[T]['returnType']) => void): void;
+        send<T extends keyof ProtocolMappingApi.Commands>(event: T, params: ProtocolMappingApi.Commands[T]['paramsType'][0],
+            callback: (returnMessage: ProtocolMappingApi.Commands[T]['returnType']) => void): void;
+        send<T extends keyof ProtocolMappingApi.Commands>(event: T, params: ProtocolMappingApi.Commands[T]['paramsType'][0], sessionId: string,
+            callback: (returnMessage: ProtocolMappingApi.Commands[T]['returnType']) => void): void;
```

Comment:
```suggestion
        send<T extends keyof ProtocolMappingApi.Commands>(event: T, callback: (error: boolean, response: ProtocolMappingApi.Commands[T]['returnType']) => void): void;
        send<T extends keyof ProtocolMappingApi.Commands>(event: T, params: ProtocolMappingApi.Commands[T]['paramsType'][0],
            callback: (error: boolean, response: ProtocolMappingApi.Commands[T]['returnType']) => void): void;
        send<T extends keyof ProtocolMappingApi.Commands>(event: T, params: ProtocolMappingApi.Commands[T]['paramsType'][0], sessionId: string,
            callback: (error: boolean, response: ProtocolMappingApi.Commands[T]['returnType']) => void): void;
```

First `callback` parameter is `boolean`:
doc: https://github.com/cyrus-and/chrome-remote-interface/blob/v0.31.0/README.md#clientsendmethod-params-sessionid-callback
code: https://github.com/cyrus-and/chrome-remote-interface/blob/v0.31.0/lib/chrome.js#L256-L260

Second `callback` parameter is complicated but I suppose `ProtocolMappingApi.Commands[T]['returnType']` will do for now.

Probably better to change the tests as well.